tests/lib/rules/no-extra-parens.js

```diff
@@ -291,7 +298,61 @@ ruleTester.run("no-extra-parens", rule, {
         { code: "foo instanceof (bar instanceof baz)", options: ["all", { nestedBinaryExpressions: false }] },
         { code: "foo in (bar in baz)", options: ["all", { nestedBinaryExpressions: false }] },
         { code: "foo + (bar + baz)", options: ["all", { nestedBinaryExpressions: false }] },
-        { code: "foo && (bar && baz)", options: ["all", { nestedBinaryExpressions: false }] }
+        { code: "foo && (bar && baz)", options: ["all", { nestedBinaryExpressions: false }] },
+
+        // ignoreJSX === "all"
```

Comment:
Minor convention nitpick: could you change these comments so that they look like this:

```js
// ["all", {ignoreJSX: "all"}]
```
That seems to be the way other "divider" comments in this file do it (See [line 213](https://github.com/eslint/eslint/pull/7926/files#diff-654d0ea8cd0d760e74e00806df5e3125R213), for example).